tests/parsers/agilent_gen5/testdata/fluorescence/endpoint_single_plate_example.txt

```diff
@@ -0,0 +1,126 @@
+B	STD1	STD1	SPL1	SPL1	SPL9	SPL9	SPL17	SPL17	SPL25	SPL25	SPL33	SPL33	SPL41	SPL41	SPL49	SPL49	SPL57	SPL57	SPL65	SPL65	SPL73	SPL73	SPL81	SPL81	Well ID
+	200	200																							Conc/Dil
+C	STD2	STD2	SPL2	SPL2	SPL10	SPL10	SPL18	SPL18	SPL26	SPL26	SPL34	SPL34	SPL42	SPL42	SPL50	SPL50	SPL58	SPL58	SPL66	SPL66	SPL74	SPL74	SPL82	SPL82	Well ID
+	100	100																							Conc/Dil
+D	STD2	STD2	SPL2	SPL2	SPL10	SPL10	SPL18	SPL18	SPL26	SPL26	SPL34	SPL34	SPL42	SPL42	SPL50	SPL50	SPL58	SPL58	SPL66	SPL66	SPL74	SPL74	SPL82	SPL82	Well ID
+	100	100																							Conc/Dil
+E	STD3	STD3	SPL3	SPL3	SPL11	SPL11	SPL19	SPL19	SPL27	SPL27	SPL35	SPL35	SPL43	SPL43	SPL51	SPL51	SPL59	SPL59	SPL67	SPL67	SPL75	SPL75	SPL83	SPL83	Well ID
+	50	50																							Conc/Dil
+F	STD3	STD3	SPL3	SPL3	SPL11	SPL11	SPL19	SPL19	SPL27	SPL27	SPL35	SPL35	SPL43	SPL43	SPL51	SPL51	SPL59	SPL59	SPL67	SPL67	SPL75	SPL75	SPL83	SPL83	Well ID
+	50	50																							Conc/Dil
+G	STD4	STD4	SPL4	SPL4	SPL12	SPL12	SPL20	SPL20	SPL28	SPL28	SPL36	SPL36	SPL44	SPL44	SPL52	SPL52	SPL60	SPL60	SPL68	SPL68	SPL76	SPL76	SPL84	SPL84	Well ID
+	25	25																							Conc/Dil
+H	STD4	STD4	SPL4	SPL4	SPL12	SPL12	SPL20	SPL20	SPL28	SPL28	SPL36	SPL36	SPL44	SPL44	SPL52	SPL52	SPL60	SPL60	SPL68	SPL68	SPL76	SPL76	SPL84	SPL84	Well ID
+	25	25																							Conc/Dil
+I	STD5	STD5	SPL5	SPL5	SPL13	SPL13	SPL21	SPL21	SPL29	SPL29	SPL37	SPL37	SPL45	SPL45	SPL53	SPL53	SPL61	SPL61	SPL69	SPL69	SPL77	SPL77	SPL85	SPL85	Well ID
+	12.5	12.5																							Conc/Dil
+J	STD5	STD5	SPL5	SPL5	SPL13	SPL13	SPL21	SPL21	SPL29	SPL29	SPL37	SPL37	SPL45	SPL45	SPL53	SPL53	SPL61	SPL61	SPL69	SPL69	SPL77	SPL77	SPL85	SPL85	Well ID
+	12.5	12.5																							Conc/Dil
+K	STD6	STD6	SPL6	SPL6	SPL14	SPL14	SPL22	SPL22	SPL30	SPL30	SPL38	SPL38	SPL46	SPL46	SPL54	SPL54	SPL62	SPL62	SPL70	SPL70	SPL78	SPL78	SPL86	SPL86	Well ID
+	6.2	6.2																							Conc/Dil
+L	STD6	STD6	SPL6	SPL6	SPL14	SPL14	SPL22	SPL22	SPL30	SPL30	SPL38	SPL38	SPL46	SPL46	SPL54	SPL54	SPL62	SPL62	SPL70	SPL70	SPL78	SPL78	SPL86	SPL86	Well ID
+	6.2	6.2																							Conc/Dil
+M	STD7	STD7	SPL7	SPL7	SPL15	SPL15	SPL23	SPL23	SPL31	SPL31	SPL39	SPL39	SPL47	SPL47	SPL55	SPL55	SPL63	SPL63	SPL71	SPL71	SPL79	SPL79	SPL87	SPL87	Well ID
+	3.1	3.1																							Conc/Dil
+N	STD7	STD7	SPL7	SPL7	SPL15	SPL15	SPL23	SPL23	SPL31	SPL31	SPL39	SPL39	SPL47	SPL47	SPL55	SPL55	SPL63	SPL63	SPL71	SPL71	SPL79	SPL79	SPL87	SPL87	Well ID
+	3.1	3.1																							Conc/Dil
+O	BLK	BLK	SPL8	SPL8	SPL16	SPL16	SPL24	SPL24	SPL32	SPL32	SPL40	SPL40	SPL48	SPL48	SPL56	SPL56	SPL64	SPL64	SPL72	SPL72	SPL80	SPL80	SPL88	SPL88	Well ID
+																									Conc/Dil
+P	BLK	BLK	SPL8	SPL8	SPL16	SPL16	SPL24	SPL24	SPL32	SPL32	SPL40	SPL40	SPL48	SPL48	SPL56	SPL56	SPL64	SPL64	SPL72	SPL72	SPL80	SPL80	SPL88	SPL88	Well ID
+																									Conc/Dil
+
+
+
+Actual Temperature:	21.6
+
+Results
+	1	2	3	4	5	6	7	8	9	10	11	12	13	14	15	16	17	18	19	20	21	22	23	24
```

Comment:
This file creates an ASM of 1.8Mb, since this is a new example and we already have plenty with large result tables, could you please reduce the sice of this matrix to 3x2? (3 columns, and keep `A` and `B` rows. Please do it with the Layout matrix too.